src/main/groovy/pl/allegro/tech/build/axion/release/domain/PredefinedVersionIncrementer.groovy

```diff
@@ -43,7 +43,7 @@ enum PredefinedVersionIncrementer {
                     .build()
             }
         }
-        return context.currentVersion.incrementPatchVersion()
+        return context.currentVersion.incrementPatchVersion('rc1')
```

Comment:
i wnoder.. this is a breaking change in increment_prerelease behaviour. one might expect, that you can have this always on and it behaves like a normal incrementer unless `rcX` is found. I would rather have a separate incrementer for adding rc1